src/browser/components/MainPage.jsx

```diff
@@ -315,12 +318,12 @@ const MainPage = createReactClass({
           transitionEnterTimeout={300}
           transitionLeaveTimeout={500}
         >
-          { (this.state.targetURL === '') ?
-            null :
+          { (defaultConfig.displayUrlOnHover === true && this.state.targetURL !== '') ?
```

Comment:
`defaultConfig` is explicitly used. So can only maintainer change the setting on source code?

Reply:
@yuya-oc yes for this implementation that's the case. You enable/disable the display of URL hover before building and dist.